skimage/future/graph/rag.py

```diff
@@ -431,28 +428,27 @@ def draw_rag(labels, rag, img, border_color=None, node_color='#ffff00',
     img : ndarray, shape (M, N, 3)
         Input image.
     border_color : colorspec, optional
-        Any matplotlib colorspec.
-    node_color : colorspec, optional
-        Any matplotlib colorspec. Yellow by default.
-    edge_color : colorspec, optional
-        Any matplotlib colorspec. Green by default.
-    colormap : colormap, optional
-        Any matplotlib colormap. If specified the edges are colormapped with
-        the specified color map.
+        Color with which the borders of regions are drawn.
+    edge_width : float, optional
+        The thickness with which the RAG edges are drawn.
+    edge_cmap : colormap, optional
+        Any matplotlib colormap with which the edges are drawn.
+    img_cmap : colormap, optional
+        Any matplotlib colormap with which the image is draw. If set to `None`
+        the image is drawn as it is.
     thresh : float, optional
-        Edges with weight below `thresh` are not drawn, or considered for color
+        Edges with weight above `thresh` are not drawn, or considered for color
         mapping.
-    desaturate : bool, optional
-        Convert the image to grayscale before displaying. Particularly helps
-        visualization when using the `colormap` option.
     in_place : bool, optional
         If set, the RAG is modified in place. For each node `n` the function
         will set a new attribute ``rag.node[n]['centroid']``.
+    ax : matplotlib axes, optional
+        The axes to draw on.
 
-    Returns
+    Reutrns
     -------
-    out : ndarray, shape (M, N, 3)
-        The image with the RAG drawn.
+    smap : matplotlib ScalarMappable
+         A ScalarMappable for the edges that can be used to show colorbars.
```

Comment:
I have no idea what a ScalarMappable is, so perhaps adding a hyperlink reference to this would be useful.